src/main/java/ua/dymohlo/jdbcpractic/JDBCMethods.java

```diff
@@ -0,0 +1,128 @@
+package ua.dymohlo.jdbcpractic;
+
+import java.sql.*;
+
+public class JDBCMethods {
+    public static void insertIntoTableContact() {
+        try (Connection connection = ConnectionManager.getConnection()) {
+            String newMailContact = "INSERT INTO alevelgraduates.contact(contact, contact_type) VALUES ('snowSecond@gmail.com', 'mail')";
+            PreparedStatement preparedStatement = connection.prepareStatement(newMailContact);
```

Comment:
PreparedStatent needed to insert values. There is no setting the parameters of the statement.
You need to create a parametrized SQL query and set appropriate parameters

Links to additional info:
https://www.geeksforgeeks.org/how-to-use-preparedstatement-in-java/
https://www.javatpoint.com/PreparedStatement-interface